cypress/integration/admin/pageBuilder/blocks/pageBlocksExportImport.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Blocks Export/Import", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllBlocks();
+        cy.pbDeleteAllBlockCategories();
+    });
+
+    const blockNames1 = ["Block1Name"];
+
+    const blockNames2 = ["Block1Name", "Block2Name"];
+
+    const blockNames3 = ["Block1Name", "Block2Name", "Block3Name"];
+
+    //Data used for creating multible block categories
+    const blockCategoryData1 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    const blockCategoryData2 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    const blockCategoryData3 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    it("Test the importation and exportation of all blocks", () => {
+        cy.visit("/page-builder/page-blocks");
+        // Exports all created data and saves the exported string value.
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData1,
+            blockNames: blockNames1
+        });
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData2,
+            blockNames: blockNames2
+        });
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData3,
+            blockNames: blockNames3
+        });
+
+        cy.findByPlaceholderText("Search blocks").should("exist");
+        cy.findByTestId("pb-blocks-list-options-menu").click();
+        cy.findByText("Export all blocks").click();
+        cy.findByText("Your export is now ready!").should("exist");
+
+        cy.get("span.link-text.mdc-typography--body2")
+            .invoke("text")
+            .then(importUrl => {
+                cy.pbDeleteAllBlockCategories();
+                cy.pbDeleteAllBlocks();
+
+                cy.visit("/page-builder/page-blocks");
+                cy.findByPlaceholderText("Search blocks").should("exist");
+                cy.findByTestId("pb-blocks-list-options-menu").click();
+                cy.findByRole("menuitem", { name: "Import blocks" }).click();
```

Comment:
I was confused here - where's the actual click on the PASTE FILE URL button? 🤔 

![image](https://github.com/webiny/webiny-js/assets/5121148/320d137e-306f-45a6-a2ae-4809f73d9e21)

Reply:
There isn't one. This is caused by both elements being part of the same div and not giving it enough time to load properly.
It is now fixed